internal/ldconfig/ldconfig.go

```diff
@@ -220,6 +227,19 @@ func createLdsoconfdFile(pattern string, dirs ...string) error {
 	return nil
 }
 
+// ensureLdsoconfFile creates a "standard" top-level ld.so.conf file if none exists.
+//
+// The created file will contain a single include statement for "`ldsoconfdDir`/*.conf".
+func ensureLdsoconfFile(topLevelLdsoconfFilePath, ldsoconfdDir string) error {
+	configFile, err := os.OpenFile(topLevelLdsoconfFilePath, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0644)
+	if err != nil && !os.IsExist(err) {
+		return fmt.Errorf("failed to create top-level ld.so.conf file: %w", err)
+	}
+	defer configFile.Close()
+	configFile.WriteString("include " + ldsoconfdDir + "/*.conf\n")
```

Comment:
One thing about this implementation is that we add this line even if the config file already exists OR already includes this line. Should we we check these conditions (or at least not modify the file if it already exists)?

Reply:
The code _accidentally_ works because if the file exists `os.OpenFile` with `os.O_CREATE|os.O_EXCL` will fail and `configFile` will be `nil`. But I will clarify the code.